presto-product-tests/bin/run_on_docker.sh

```diff
@@ -46,7 +48,7 @@ function check_presto() {
   run_in_application_runner_container \
     java -jar "/docker/volumes/presto-cli/presto-cli-executable.jar" \
     ${CLI_ARGUMENTS} \
-    --execute "SHOW CATALOGS" | grep -i hive
+    --execute "SHOW CATALOGS" | grep -iq hive
```

Comment:
unrelated change?

Reply:
Thanks to that change we `Avoid meaningless output from grep command`. But from the another grep command.